scripts/shared_functions.sh

```diff
@@ -47,8 +63,8 @@ reload_tmux_environment() {
 plugin_already_installed() {
 	local plugin="$1"
 	local plugin_path="$(shared_plugin_path "$plugin")"
-	cd "$plugin_path" &&
-		git remote >/dev/null 2>&1
+	cd "$plugin_path" #&& unmarry from git
```

Comment:
In this case we could replace the wholy thing with an `if [ -d "$plugin_path" ]`, right?